src/_main_/utils/common.py

```diff
@@ -261,10 +261,15 @@ def extract_location(args):
     return location
 
 
+# def is_value(b):
+#     if b and b != "undefined" and b != "NONE":
+#         return True
+#     if b == "":  # an empty string is a string value
+#         return True
+#     return False
+
 def is_value(b):
-    if b and b != "undefined" and b != "NONE":
-        return True
-    if b == "":  # an empty string is a string value
+    if b and b not in ["undefined", "null", "None", ""]:
```

Comment:
@abdullai-t - this is not correct.  See the comment above, where "" should return True.  Also note that the empty string tests as b=false, so maybe your change doesn't break anything but it doesn't do as you intend.